### DISCUSSION TITLE
Discord invitation open to all

### DISCUSSION THREAD

**Discussion by ceddybi** [stoqey/ib#26] [General]
DISCORD: https://discord.gg/T4VjBrqGtK

anyone on here, please join our discord for quicker communication

Thank you very much 🙏

**Comment by mfrener** [comment]
can you add @Tsopic pls? looks like we have one more maintainer for IB API :)
edit: ohh.. wait.. we need to join I think

**Reply by ceddybi** [reply]
Thank you @mfrener, just added @Tsopic to the team as well, you're super awesome, you rock 🙏🤘🙌💯 @mfrener @Tsopic

**Reply by Tsopic** [reply]
Thanks, I still don't have write permission to the repo.

**Comment by ceddybi** [comment]
+ @netshade
